src/main/java/com/google/sps/classes/Utility.java

```diff
@@ -18,17 +18,45 @@
 import com.google.appengine.api.users.UserServiceFactory;
 import com.google.gson.Gson;
 import com.google.sps.classes.SqlConstants;
+import com.google.sps.classes.Utility;
 import java.sql.*;
 import java.util.logging.Level;
 import java.util.logging.Logger;
+import javax.servlet.http.HttpServletRequest;
+import javax.sql.DataSource;
 
 // TODO(aabundis): Add JUnit tests for utility functions.
 
 /**
  * Utility methods used across classes. Just import class to access all methods.
  */
 public final class Utility {
-  // TODO(oumontiel): Move constants to different file.
+  // Define if running locally or deploying the current branch.
+  // Define IS_LOCALLY_DEPLOYED constant as true for a local deployment or deploy for a cloud deployment.
+  public static final boolean IS_LOCALLY_DEPLOYED = true;
```

Comment:
This should be defaulted to 'false' in Master though, right? Because Master should reflect the deployed version.